service/iam/model.go

```diff
@@ -478,7 +478,7 @@ func (f *PatchOp) Type() string {
 
 type PatchSchema string
 
-const PatchSchemaUrnIetfParamsScimApiMessagesPatchop PatchSchema = `urn:ietf:params:scim:api:messages:2.0:PatchOp`
+const PatchSchemaUrnIetfParamsScimApiMessages20PatchOp PatchSchema = `urn:ietf:params:scim:api:messages:2.0:PatchOp`
```

Comment:
single-value enums hint at patch operations for SCIM having to be manually coded

Reply:
+1. Until this is supported at the OpenAPI level, is there a practical approach we can take?

Reply:
I see that there is an x-databricks-alias-enum field which we can use to simplify this name significantly, I'll do that in a separate PR since that requires an OpenAPI change in universe.

Reply:
@mgyucht these enums won't be present in Java SDK - it won't allow non-alphanum values. 

one of the options is an annotation for a field to always be present in the code. still - it's only for SCIM `patch` requests. i think it's best to hand-code all patch requests...

Reply:
Yeah, we'll have to figure out how best to expose this in the API. It's a very difficult protocol to generate an SDK for... I used jackson's at-JsonProperty annotation to control serde for these fields, so I think it should be OK in Java.

Reply:
but java enums don't support non-alphanum characters.

Reply:
You mean the enum value names? They just need to be valid identifiers IIUC, which are alphanum + underscore and dollar sign. See https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-Identifier

> The "Java letters" include uppercase and lowercase ASCII Latin letters A-Z (\u0041-\u005a), and a-z (\u0061-\u007a), and, for historical reasons, the ASCII underscore (_, or \u005f) and dollar sign ($, or \u0024). The $ sign should be used only in mechanically generated source code or, rarely, to access pre-existing names on legacy systems.

I don't see any errors from IntelliJ with this change, and there are examples where underscores are allowed in enum values: https://stackoverflow.com/questions/9712977/is-it-possible-to-have-an-enum-class-with-enums-of-two-or-more-words.